src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,37 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/driver/car/add")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/drivers/cars/add")
```
It is good practice to use plurals.
And don't forget to change this link in jsp